frontend/user-dashboard/src/lib/responseEnvelope.ts

```diff
@@ -1,84 +0,0 @@
-import { error, redirect } from '@sveltejs/kit';
-
-export class ResponseEnvelope {
```

Comment:
This is now provided in the `frontend-toolkit` package under the `ApiResponse` structure. The goal is to provide a base class that any frontend service can then reuse: we benefit from defining it only once and in case of changes it can easily be rolled out in a gradual way to the frontend services as we see fit.